<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CWidgetHttp</class>
 <widget class="QWidget" name="CWidgetHttp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <widget class="QPushButton" name="btnLoad">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>下载</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>URL：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditURL">
       <property name="text">
        <string>http://192.168.43.159:2580/XSendfile/RjovSUFMaWdodFBhdGgvMTUxMDM4NDAwMDE3MjM2NjA0LzIwMTkxMDIzL0UwQzdFMTQ4OEIyQzRERTY4MjBBRjVERkQ3NTUwODI5LzIwMTkxMDIzMTA0MTU1X1VQMDI5NF9FMEM3RTE0ODhCMkM0REU2ODIwQUY1REZENzU1MDgyOS5qcGc=</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>本地保存路径：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditLocalPath">
       <property name="text">
        <string>C:\Users\aa\Desktop\123.jpg</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QTextEdit" name="textEditLog"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
